scaling_performance/cluster_limits.adoc

```diff
@@ -0,0 +1,102 @@
+[[scaling-performance-cluster-limits]]
+= Cluster Limits
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+This topic summarizes the limits for objects in {product-title}.
+
+In most cases, exceeding these thresholds results in lower overall performance.
+It does not necessarily mean that the cluster will fail over.
+
+Some of the limits represented in this topic are given for the largest possible
+cluster. For smaller clusters, the limits are proportionally lower.
+
+There are many factors that influence the stated thresholds, including the etcd
+version or storage data format.
+
+[[scaling-performance-current-cluster-limits]]
```

Comment:
One piece of additional feedback is to rename "Current Limits" with the precise version number.  My intent is to update this page with each release with new columns, even if none of the values themselves change.  So "Current" is ambiguous, please make it 3.7.